Backendless/Data/IDataStore.cs

```diff
@@ -123,6 +127,8 @@ public interface IDataStore<T>
     Task<T> FindByIdAsync( T entity, IList<string> relations, int? relationsDepth );
   #endif
     void FindById( string id, AsyncCallback<T> responder );
+    void FindById( string id, DataQueryBuilder queryBuilder, AsyncCallback<T> responder );
+    void FindById( T entity, DataQueryBuilder queryBuilder, AsyncCallback<T> responder );

```

Comment:
does this method override cause a conflict with the method which accepts string?

Reply:
@markpiller no